File: engine/engine-model/src/main/scala/tools/packet/cashshop/operation/ShowBoughtInventorySlots.scala
package tools.packet.cashshop.operation

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ShowBoughtInventorySlots(private var _inventoryType: Int, private var _slots: Short) extends PacketInput {
  def inventoryType: Int = _inventoryType

  def slots: Short = _slots

  override def opcode(): SendOpcode = SendOpcode.CASH_SHOP_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/cashshop/operation/ShowBoughtInventorySlots.java
package tools.packet.cashshop.operation;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ShowBoughtInventorySlots(int inventoryType, short slots) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.CASH_SHOP_OPERATION;
   }
}